src/hooks/useArticlePreviewTracking.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'preact/hooks'
+import { sendEvent, isInstrumentationEnabled } from 'utils'
+
+const SCHEMA_NAME = 'VirtualPageView'
+const SCHEMA_REV = 19883675
+
+export const useArticlePreviewTracking = (
+  page, source, language
+) => {
+  if (!isInstrumentationEnabled()) {
+    return
+  }
+
+  if (page === undefined || source === undefined) {
```

Comment:
When is this called without `page` or `source`?

Reply:
_Just in case `useArticle()` is being called before the `useArticle( x, y )`, to make sure it returns empty {} instead of error._